### DISCUSSION TITLE
Simulating seasonal LAI profile of Sugarcane makes no sense

### DISCUSSION THREAD

**Discussion by ysharaby1102** [APSIMInitiative/ApsimX#9540] [Q&A]
Hi all,

I have simulated the seasonal LAI profile of sugarcane using two soils and one met file sampled from Palm Beach County, Florida, for the last 3.5 years (the clock starts 01-Oct-20 and ends 01-Mar-24). I got a surprising curve that makes no sense to me (attached), as it reduces back to zero at the end of the season at once and presents significant in-season sharp decreases in LAI.

![LAI_Sugarcane_Florida](https://github.com/user-attachments/assets/815531cb-498f-4280-934f-3231d388d1e5)

I have used the default SowingRule parameters: 

- planting date 1-oct
- duration of plant crop 450 days
- stalk density per square meter of 10 (both plant crop and ratoon)
- planting variety q124
- planting depth 150mm
- 4 ratoons before EndCrop
- duration of ratoon crop 390 days

No matter which parameter I am changing and how this surprising LAI profile does not change. Moreover, I have the actual LAI profile of sugarcane fields in Palm Beach, Florida for that time frame, as measured from Sentinel-2. They present a constant, logical LAI profile, that features a max LAI of 3 - 3.5 during late August, a min LAI of ~0.5 during late February, and a continuous trend that does not zero at once.

Any ideas what could cause this, and how to calibrate the simulation to create a more logical LAI profile?
Thanks!

**Comment by hut104** [comment]
Hi
It is difficult to debug just using the screen capture, but the nature of the time course might suggest transient water stress.  Output, check and resolve issues for water, temperature and N stress factors.  This will help deal with drivers of within season variation.
Check the summary file to see what is happening on days with sudden changes in LAI.  It may be harvesting.

**Comment by BrianCollinss** [comment]
Well LAI is supposed to be reduced to zero at the end of each crop as its harvested and a ratoon crop starts regrowing.

As for the in-season drops, it might be due to lodging if it occurs on one day. Check out if it's been activated and triggered.
